charts/keess/templates/cluster-role-binding.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ template "keess.fullname" . }}
+  labels:
+    {{- include "keess.labels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ template "keess.fullname" . }}
+subjects:
+- kind: ServiceAccount
+  name: {{ template "keess.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+{{- end }}
```

Comment:
We should move Clusterroles/Rolebindings to PAC, no?

Reply:
We have the purpose to make keess public soon, so considering that the idea is to consolidate the deployment of all artifacts inside of this chart.

Reply:
Right that makes sense -- we may run into issues during deployment.

Reply:
I already tested the deployment on alpha and beta clusters. No issues until now.